### ISSUE TITLE
http://doc.goclouder.net/ is not working

### ISSUE THREAD

**Issue by lucode** [clouder-community/clouder#125]
http://doc.goclouder.net/ does only show nginx.

**Comment by YannickB** [comment]
Yep I just saw it, I changed the CNAME of the domain, let's wait the DNS refresh

**Comment by lasley** [comment]
CNAME refresh seems to have done it, site is online

**Comment by YannickB** [comment]
Yep it looks ok, closing the issue

**Comment by pasgou** [comment]
to be re-opened?

**Comment by YannickB** [comment]
Yes, I am currently migrating my infra and it looks like the new DNS server wasn't taken into account. I expect some error this week.

**Comment by YannickB** [comment]
Ok I think the problem is resolved. If someone else manage to access goclouder.net and doc.goclouder.net, please close the ticket.

**Comment by lasley** [comment]
doc.goclouder.net is working on my end

goclouder.net is yielding a security error - `ERR_SPDY_INADEQUATE_TRANSPORT_SECURITY`

**Comment by lasley** [comment]
May want to look into upgrading your Nginx & using http2 instead of spdy

**Comment by pasgou** [comment]
it's working this morning.

doc is redirected to http://clouder.readthedocs.io/en/master/

Is master operationnal?
